src/main/java/com/example/bulletinboard/config/WebSecurityConfig.java

```diff
@@ -18,7 +19,10 @@ public class WebSecurityConfig {
             "/v3/api-docs",
             "/webjars/**",
             "/login",
-            "/register"
+            "/register",
+            "/ads/**",
```

Comment:
```suggestion
            "/ads",
```
только /ads, а не все запросы на /ads/**